.gitmodules

```diff
@@ -1,4 +1,3 @@
 [submodule "vendor/quantlib"]
 	path = vendor/quantlib
-	url = https://github.com/lballabio/QuantLib.git
-	branch = v1.38
\ No newline at end of file
+	url = https://github.com/lballabio/QuantLib.git
```

Comment:
Why did you remove the `branch = v1.38`? You don't need to use it to make sure that the submodule doesn't go out of sync?